packages/parser-package-json/src/types.ts

```diff
@@ -0,0 +1,27 @@
+import { Event, ErrorEvent, Events } from 'hint/dist/src/lib/types/events';
+import { IJSONLocationFunction, ISchemaValidationError } from 'hint/dist/src/lib/types';
+
+export type PackageJsonInvalidJSON = ErrorEvent;
+
+/** Data type sent when the parse starts parsing */
+export type PackageJsonParseStart = Event;
+
+/** The object emitted by the `package-json` parser */
+export type PackageJsonParsed = Event & {
+    /** The package json parsed */
+    config: any;
```

Comment:
We haven't done this before, but want to try something like [json-schema-to-typescript](https://github.com/bcherny/json-schema-to-typescript) to automatically create some TypeScript types to reference here instead of `any`?

We've chatted about going in the other direction in the past in webhintio/rfcs#43, but I think since we have a number of externally sourced schemas, using the schema itself as the "source of truth" may be better.